tests/ztest/error_hook/src/main.c

```diff
@@ -307,7 +311,7 @@ void test_main(void)
 			 ztest_user_unit_test(test_catch_assert_fail),
 			 ztest_user_unit_test(test_catch_fatal_error),
 			 ztest_unit_test(test_catch_assert_in_isr),
-			 ztest_user_unit_test(test_catch_z_oops)
+			 ztest_unit_test(test_catch_z_oops)
```

Comment:
HI,  this ztest_user_unit_test() and ztest_unit_test() both work here, I guess it might be some reason for change this. could you please let me know, thank you!